modules/services/mopidy.nix

```
@@ -97,6 +97,16 @@ in {
         more details.
       '';
     };
+
+    extraConfigFiles = mkOption {
+      default = [ ];
+      type = types.listOf types.str;
```

I think you always want an absolute path, so can do a bit better than `str`:
```suggestion
      type = types.listOf types.path;
```